include/iarc7_motion/LandPlanner.hpp

```diff
@@ -68,15 +68,19 @@ class LandPlanner
 
     LandState state_;
 
+
```

Comment:
I don't like this

Reply:
I agree. Too much white space

Reply:
![image](https://user-images.githubusercontent.com/4535296/41270140-88408bae-6dd7-11e8-9010-ce138df27533.png)